custom_components/zaptec/switch.py

```diff
@@ -53,7 +53,7 @@ def _update_from_zaptec(self) -> None:
         """Update the entity from Zaptec data."""
         # Called from ZaptecBaseEntity._handle_coordinator_update()
         state = self._get_zaptec_value()
```

Comment:
```suggestion
        state = self._get_zaptec_value().lower()
```
Are we likely to get any cases where we have a str-value that shouldn't be converted to lower case, or should we consider including the conversion to lower inside  the _get_zaptec_value() function?

Reply:
Good question. We might have string values, such as name or address that we'd like to preserve as is, so `_get_zaptec_value()` can't do it for all. But perhaps an argument `preserve_str=True` could be used for those cases where we do want to preserve it?

Reply:
That sounds like a good idea. On a related note, the states of `installation_type` and `device_type` are currently not translated (except Unknown?), so either they need to all be translated, or we need to preserve those.

Reply:
For these particular ones, yes, I think it doesn't make much sense to translate them.